src/main/groovy/com/benjaminsproule/swagger/gradleplugin/GenerateSwaggerDocsTask.groovy

```diff
@@ -42,13 +42,13 @@ class GenerateSwaggerDocsTask extends DefaultTask {
     @TaskAction
     generateSwaggerDocuments() {
         def swaggerExtension = project.extensions.getByName(SwaggerExtension.EXTENSION_NAME)
-        ClassFinder.createInstance(project)
-        ResourceFinder.createInstance(project)
 
         ensureCompatibleSwaggerSpec()
 
         try {
             for (ApiSourceExtension apiSourceExtension : swaggerExtension.apiSourceExtensions) {
+                ClassFinder.createInstance(project)
```

Comment:
The actual problem is with the caching, not with the actual `ClassFinder`, so shouldn't this be a call to clear the cache rather than recreating the object?